src/Microsoft.AspNet.Mvc.Core/Formatters/DelegatingEnumerable.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.AspNet.Mvc
```

Comment:
Is there a good namespace to put these in? I don't see a lot of users reaching for this code frequently